src/EditorFeatures/Core/Implementation/FindReferences/FindReferencesCommandHandler.cs

```diff
@@ -51,12 +52,16 @@ internal void FindReferences(ITextSnapshot snapshot, int caretPosition)
                     var service = document.Project.LanguageServices.GetService<IFindReferencesService>();
                     if (service != null)
                     {
-                        if (!service.TryFindReferences(document, caretPosition, context))
+                        var globalOperationService = document.Project.Solution.Workspace.Services.GetService<IGlobalOperationNotificationService>();
+                        using (globalOperationService?.Start("FindReferences"))
```

Comment:
the string will pass down to listener. currently there is no one care though. probably should log that information to etl.